cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -76,19 +79,20 @@
   private final Supplier<EndpointStrategy> endpointStrategySupplier;
   private final Id.Namespace namespaceId;
   private final HttpRequestConfig httpRequestConfig;
+  private final boolean securityEnabled;
 
   DatasetServiceClient(final DiscoveryServiceClient discoveryClient, Id.Namespace namespaceId,
-                              CConfiguration cConf) {
+                       CConfiguration cConf) {
     this.endpointStrategySupplier = Suppliers.memoize(new Supplier<EndpointStrategy>() {
       @Override
       public EndpointStrategy get() {
         return new RandomEndpointStrategy(discoveryClient.discover(Constants.Service.DATASET_MANAGER));
       }
     });
     this.namespaceId = namespaceId;
-
     int httpTimeoutMs = cConf.getInt(Constants.HTTP_CLIENT_TIMEOUT_MS);
     this.httpRequestConfig = new HttpRequestConfig(httpTimeoutMs, httpTimeoutMs);
+    this.securityEnabled = cConf.getBoolean(Constants.Security.ENABLED);
```

Comment:
The ``AuthorizationInstantiator`` logs a warning that even though authorization is enabled, it will be no-op because security is disabled - and we use a NoOpAuthorizer